src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -36,7 +36,12 @@ import { OperationType } from 'components/TransactionConfirmationModal'
 import { ONE_HUNDRED_PERCENT } from 'constants/misc'
 
 const ErrorMessages = {
-  InsufficientBalance: (symbol = '') => `Insufficient ${symbol} balance to cover investment amount`,
+  NoBalance: (symbol = '') => `You don't have ${symbol} balance to invest`,
+
+  InsufficientBalance: (selfClaim: boolean, symbol = '') =>
+    selfClaim
+      ? `Insufficient ${symbol} balance to cover investment amount`
+      : `Your ${symbol} balance is not enough to cover 100% of the investment amount.`,
```

Comment:
Since you are calling them in different places (rather than just passing the self claim flag) I think it makes more sense to have 2 different entries than having a conditional inside of it

Reply:
Fine with me! will do